example-deployment/k8s/nodeserver.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: node-app-service
+spec:
+  selector:
+    app: node-app 
+  type: LoadBalancer
+  ports:
+  - protocol: TCP
+    port: 9009
+    targetPort: 9009
+    nodePort: 31110
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: node-app-deployment
+  labels:
+    app: node-app
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: node-app
+  template:
+    metadata:
+      labels:
+        app: node-app 
+    spec:
+      containers:
+      - name: node-app
+        imagePullPolicy: Never # remove this in a real deployment
+        image: ekumen/node-web-app:latest
```

Comment:
Is this a private repo in dockerhub? I can't find it in the public list and it's not built locally.

Reply:
Removed the yaml file